integration_tests/src/main/python/join_test.py

```diff
@@ -399,7 +399,6 @@ def do_join(spark):
 
 @ignore_order(local=True)
 @pytest.mark.parametrize('rows', ['(1)', '(1), (null)', '()'], ids=['no_nulls', 'has_nulls', 'empty'])
-@allow_non_gpu(*non_utc_allow)
```

Comment:
This is to address the comment at https://github.com/NVIDIA/spark-rapids/pull/13691/#discussion_r2490895813